pkg/controller/syncer/keycloak.go

```diff
@@ -201,9 +208,34 @@ func (k *KeycloakSyncer) Sync() ([]userv1.Group, error) {
 			return nil, err
 		}
 
+		chidlrenGroups := []string{}
+
+		for _, subgroup := range cachedGroup.SubGroups {
+			chidlrenGroups = append(chidlrenGroups, *subgroup.Name)
+		}
+
+		parentGroups := []string{}
+
+		for _, group := range k.CachedGroups {
+			for _, subgroup := range group.SubGroups {
+				if subgroup.Name == cachedGroup.Name {
+					parentGroups = append(parentGroups, *group.Name)
+				}
+			}
+		}
+
 		// Set Host Specific Details
 		ocpGroup.GetAnnotations()[constants.SyncSourceHost] = url.Host
 		ocpGroup.GetAnnotations()[constants.SyncSourceUID] = *cachedGroup.ID
+		if len(chidlrenGroups) > 0 {
+			ocpGroup.GetAnnotations()["hierarchy_children"] = strings.Join(chidlrenGroups, ",")
```

Comment:
Can you add the annotation keys to the `constants.go` file?

Reply:
done